moveit_ros/warehouse/warehouse/src/broadcast.cpp

```diff
@@ -68,6 +68,7 @@ int main(int argc, char** argv)
   // time to wait in between publishing messages
   double delay = 0.001;
 
+  // clang-format off
```

Comment:
I'm not a huge fan of putting this here, but there was insane indentation. clang-format-12 really acts weird on 20.04 and clang-format-10 is not supported in Ubuntu 22.04. In fact I am guessing this weirdness is because our format jobs are running in 20.04 images and I'm formatting in 22.04 - which seemed reasonable. Unfortunately GH ubuntu-latest is still 20.04 and there is no option for 22.04, I believe until May. I might migrate our pre-commit to run on an 22.04 image to avoid weird pre-commit problems.